identity-tools/developer_identity/generate_self_signed_certs.sh

```diff
@@ -45,7 +45,7 @@ _createDevicePrivateKey() {
 }
 
 _createDeviceCertificate() {
-	(echo '[ req ]'; echo 'distinguished_name=dn'; echo 'prompt = no'; echo '[ dn ]'; echo 'C=US'; echo 'ST=Texas';echo 'L=Austin';echo 'O=ARM';echo "OU=$OU";echo "CN=$internalid";) > ${cert_dir}/device.cnf
+	(echo '[ req ]'; echo 'distinguished_name=dn'; echo 'prompt = no'; echo '[ dn ]'; echo 'C=US'; echo 'ST=Texas';echo 'L=Austin';echo 'O=ARM';echo "OU=$ACCOUNT_ID";echo "CN=$DEVICE_ID";) > ${cert_dir}/device.cnf
```

Comment:
Could also be done with a HEREDOC.

```
cat << EOF > "${cert_dir}/device.cnf"
Text
Goes
$variable_to_insert
Here
EOF```

Reply:
Good to know. I will make this change the next time we make any modification to this script.